Src/FluentAssertions/Specialized/ActionAssertions.cs

```diff
@@ -9,7 +10,12 @@ namespace FluentAssertions.Specialized
     [DebuggerNonUserCode]
     public class ActionAssertions : DelegateAssertions<Action>
     {
-        public ActionAssertions(Action subject, IExtractExceptions extractor) : base(subject, extractor)
+        public ActionAssertions(Action subject, IExtractExceptions extractor) : this(subject, extractor, new Clock())
+        {
+            Subject = subject;
```

Comment:
`Subject` is set in the other constructor?

Reply:
@dennisdoomen this one

Reply:
Apparently I forgot to push my changes....